src/Util/MailerLoggerUtil.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Util;
+
+use Symfony\Bridge\Monolog\Logger;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Swift_Events_SendEvent;
+use Swift_Events_SendListener;
+
+/**
+ * A utility to log Swift e-mails.
+ */
+class MailerLoggerUtil implements Swift_Events_SendListener
+{
+     /**
+      * A Monolog logger.
+      *
+      * @var Logger
+      */
+    protected $logger;
+
+    /**
+     * MailerLoggerUtil constructor.
+     *
+     * @param LoggerInterface $logger The Logger Interface.
+     */
+    public function __construct(LoggerInterface  $logger)
```

Comment:
We don't need to use Psr\Log\LoggerInterface, monolog already implements that in its class. So we can just create a Monolog logger object, like how we have done in the Consumers, etc

Reply:
Somewhere i'm thinking one of these ways is deprecated.

Reply:
This is how it's documents in Symfony 5.0:
https://symfony.com/doc/current/components/console/logger.html